conda-store-server/tests/test_actions.py

```diff
@@ -15,9 +15,17 @@ def test_action_decorator():
     def test_function(context):
         print("stdout")
         print("stderr", file=sys.stderr)
-        context.run(["echo", "subprocess"])
-        context.run("echo subprocess_stdout", shell=True)
-        context.run("echo subprocess_stderr 1>&2", shell=True)
+        if sys.platform == "win32":
+            # echo is not a separate program on Windows
+            context.run(["cmd", "/c", "echo subprocess"])
+            context.run("echo subprocess_stdout", shell=True)
+            context.run("echo subprocess_stderr>&2", shell=True)
+            context.run("echo subprocess_stderr_no_redirect>&2", shell=True, redirect_stderr=False)
+        else:
+            context.run(["echo", "subprocess"])
+            context.run("echo subprocess_stdout", shell=True)
+            context.run("echo subprocess_stderr 1>&2", shell=True)
+            context.run("echo subprocess_stderr_no_redirect 1>&2", shell=True, redirect_stderr=False)
```

Comment:
Only the first command needs to be inside if-else.

Reply:
Nope, this is parsed slightly differently on Windows. Whitespace would be included into the output with redirect_stderr. So this is fine.